app/controllers/users_controller.rb

```diff
@@ -40,7 +40,10 @@ def update
     set_current_tab(params["user"]["tab"])
 
     if @user.update(permitted_attributes(@user))
-      RssReaderFetchUserWorker.perform_async(@user.id) if @user.feed_url.present?
+      # NOTE: [@rhymes] this queues a job to fetch the feed each time the profile is updated, regardless if the user
+      # explicitly requested "Feed fetch now" or simply updated any other field
```

Comment:
I added this comment mostly for explanatory reasons, I didn't know we were fetching the feed each time until I set out to add the feature flag below.

I think we can split it out to its own controller action but not in this PR